guided-match/ddl.sql

```diff
@@ -0,0 +1,207 @@
+   outcome_evaluator_code           VARCHAR(10) NOT NULL, 
+   journey_answer_id                INTEGER NOT NULL,
+   lot_number                       VARCHAR(10) NOT NULL        
+);
+*/
+
+
+
+/* Neo4J
+CREATE TABLE question_definitions (
+  question_id                       INTEGER PRIMARY KEY,
+  question_type                     VARCHAR(20) NOT NULL,
+  question_description              VARCHAR(2000) NOT NULL,
+  question_text                     VARCHAR(200) NOT NULL,
+  published_status                  CHAR(1) NOT NULL, -- Y(es) or N(o),
+  validate_via_agreement            CHAR(1) NOT NULL, -- Y(es) or N(o),
+);
+*/
+/* Neo4J
+CREATE TABLE question_groups (
+  question_group_id                 INTEGER PRIMARY KEY,
+  question_group_name               VARCHAR(100) NOT NULL,
+  published_status                  CHAR(1) NOT NULL -- Need to check if we don't want to make this boolean
+);
+*/
+/* Neo4J
+CREATE TABLE question_group_members (
+  question_group_member_id          INTEGER PRIMARY KEY,
+  question_group_name               VARCHAR(200) NOT NULL,
+  published_status                  CHAR(1) NOT NULL-- Y(es) or N(o),
+);
+*/
+/* Neo4J
+CREATE TABLE question_outcome_evaluators (
+  outcome_evaluator_code            VARCHAR(10) PRIMARY KEY,
+  outcome_evaluator_description     VARCHAR(200) NOT NULL
+);
+*/
+
+CREATE TABLE search_domains (
+  domain_modifier_id               INTEGER PRIMARY KEY,
+  search_id                        INTEGER NOT NULL,
+  journey_id                       UUID NOT NULL,
+  modifier_journey_name            VARCHAR(20),
+  modifier_journey_description     VARCHAR(2000)
+);
+
+create index SEDO_IDX1 ON SEARCH_DOMAINS(search_id);
+create index SEDO_IDX2 ON SEARCH_DOMAINS(journey_id);
+create index SEDO_IDX3 ON SEARCH_DOMAINS(modifier_journey_name);
+
+CREATE TABLE search_terms (
+  search_id                        INTEGER PRIMARY KEY,
+  search_term                      VARCHAR(50) NOT NULL
+);
+
+CREATE INDEX SETE_IDX1 ON SEARCH_TERMS (search_term);
+
+/* Neo4J
+CREATE TABLE standard_answers  (
+  standard_answer_id                INTEGER PRIMARY KEY,
+  valid_value                       VARCHAR(200),
+  answer_label                      VARCHAR(100),
+);
+*/
+
+/* Referential constraints */
+
+ALTER TABLE journey_instance_questions 
+ADD CONSTRAINT journey_instance_questions_journey_instances_fk FOREIGN KEY (journey_instance_id) 
+    REFERENCES journey_instances (journey_instance_id); 
+    
+ALTER TABLE journey_instance_questions 
+ADD CONSTRAINT journey_instance_questions_journey_questions_fk FOREIGN KEY (journey_question_id) 
+    REFERENCES journey_questions (journey_question_id);
+    
+ALTER TABLE search_domains
+ADD CONSTRAINT search_domains_search_terms_fk FOREIGN KEY (search_id) 
+    REFERENCES search_terms(search_id);    
+    
+ALTER TABLE search_domains
+ADD CONSTRAINT search_domains_journeys_fk FOREIGN KEY (journey_id) 
+    REFERENCES search_terms(journey_id);   

```

Comment:
[Code: 0, SQL State: 42703]  ERROR: column "journey_id" referenced in foreign key constraint does not exist

Reply:
Changed refrence to journeys from search_terms to journeys.